packages/target/README.md

```diff
@@ -0,0 +1,414 @@
+
+`@adobe/react-native-aeptarget` is a wrapper around the iOS and Android [AEP Target SDK](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target) to allow integration with React Native applications.
+
+## Peer Dependencies
+
+The Adobe Experience Platform Target extension has the following peer dependency, which must be installed prior to installing the target extension:
+
+- [Core](../core/README.md)
+- [Edge](../edge/README.md)
+
+## Installation
+
+See [Requirements and Installation](https://github.com/adobe/aepsdk-react-native#requirements) instructions on the main page
+
+Install the `@adobe/react-native-aeptarget` package:
+
+NPM:
+
+```bash
+npm install @adobe/react-native-aeptarget
+```
+
+Yarn:
+
+```bash
+yarn add @adobe/react-native-aeptarget
+```
+
+## Usage
+
+### Initializing and registering the extension
+
+Initialization of the SDK should be done in native code, documentation on how to initialize the SDK can be found [here](https://github.com/adobe/aepsdk-react-native#initializing).
+
+Example:
+
+iOS
+
+```objectivec
+@import AEPCore;
+@import AEPLifecycle;
+@import AEPEdge;
+@import AEPTarget;
+
+@implementation AppDelegate
+-(BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+  [AEPMobileCore setLogLevel: AEPLogLevelTrace];
+  [AEPMobileCore registerExtensions: @[AEPMobileEdge.class, AEPMobileTarget.class] completion:^{
+    [AEPMobileCore configureWithAppId:@"yourAppID"];
+    [AEPMobileCore lifecycleStart:@{@"contextDataKey": @"contextDataVal"}];
+  }
+  ];
+  return YES;
+}
+@end
+```
+
+Android
+
+```java
+import com.adobe.marketing.mobile.AdobeCallback;
+import com.adobe.marketing.mobile.InvalidInitException;
+import com.adobe.marketing.mobile.Lifecycle;
+import com.adobe.marketing.mobile.LoggingMode;
+import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.Edge;
+import com.adobe.marketing.mobile.Target;
+
+...
+import android.app.Application;
+...
+public class MainApplication extends Application implements ReactApplication {
+  ...
+  @Override
+  public void on Create(){
+    super.onCreate();
+    ...
+    MobileCore.setApplication(this);
+    MobileCore.setLogLevel(LoggingMode.DEBUG);
+    try {
+      Edge.registerExtension();
+      Target.registerExtension();
+      MobileCore.configureWithAppID("yourAppID");
+      MobileCore.start(new AdobeCallback() {
+        @Override
+        public void call(Object o) {
+          MobileCore.lifecycleStart(null);
+        }
+      });
+    } catch (InvalidInitException e) {
+      ...
+    }
+  }
+}
+```
+
+### Importing the extension:
+
+```typescript
+import {
+  Target,
+  TargetOrder,
+  TargetParameters,
+  TargetPrefetchObject,
+  TargetProduct,
+  TargetRequestObject
+} from '@adobe/react-native-aeptarget';
+```
+
+## API Reference
+
+#### Getting the extension version:
```

Comment:
Can we have title same format as  the other extensions?
e.g ### Getting the extension version:
Same for other APIs.